nginx/nginx.conf.template

```diff
@@ -0,0 +1,70 @@
+server {
+    listen 80;
+    server_name $NGINX_MAIN_DOMAIN;
+
+    location / {
+        return 301 https://$host$request_uri;
+    }
+}
+
+
+server {
+    listen 443 ssl;
+
+    ssl_certificate /etc/letsencrypt/live/$NGINX_MAIN_DOMAIN/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/$NGINX_MAIN_DOMAIN/privkey.pem;
+
+    server_name $NGINX_MAIN_DOMAIN;
+
+    location /api/ {
+        add_header Access-Control-Allow-Origin *;
+        proxy_pass  http://api:80/;
+    }
+
+    location /api {
+        rewrite ^ /api/ permanent;
+    }
+
+    location /manager_dashboard/ {
+        proxy_pass  http://manager_dashboard:80/;
+    }
+
+    location /manager_dashboard {
+        rewrite ^ /manager_dashboard/ permanent;
+    }
+
+    location / {
+        rewrite ^ /manager_dashboard/ permanent;
+    }
```

Comment:
this needs to be updated and should use the new manager dashboard and mapswipe dashboard

Reply:
This has been changed. Can you re-check this new one?
https://github.com/mapswipe/python-mapswipe-workers/blob/0d013617a60b747643ae21bd9b42fbff295fd687/nginx/nginx.conf.template#L11-L27